src/transformers/pipelines/zero_shot_audio_classification.py

```diff
@@ -131,8 +132,8 @@ def _forward(self, model_inputs):
         outputs = self.model(**text_inputs, **model_inputs)
 
         model_outputs = {
-            "candidate_label": candidate_labels,
-            "logits_per_audio": outputs.logits_per_audio,
+            "candidate_labels": candidate_labels,
+            "logits": outputs.logits_per_audio,
```

Comment:
not against this, but is this necessary to fix the issue we have?

Reply:
it is, afterward we `pop` both arguments and there was a typo here.